BangazonAPI/Controllers/CustomerController.cs

```diff
@@ -30,7 +30,7 @@ public SqlConnection Connection
             }
 
             [HttpGet]
-            public async Task<IActionResult> Get(string include, string q)
+            public async Task<IActionResult> Get(string include, string q, string active)
```

Comment:
Let's change `active` to a boolean rather than a string!

Reply:
I had it as a boolean and it was always resulting in active==false being true?  Which is why I changed it to be a string.  Maybe I can change it to that and show you what I mean.